### PR TITLE
Form project

### PR DESCRIPTION
**Pull request by Psiale**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```
Comment (suretrust) [inline .vscode/settings.json]: This should not be in your repository.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="css/style.css" />
+    <title>Document</title>
```
Comment (suretrust) [inline index.html]: `[Optional]` A more descriptive title won't hurt.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+      <header>
+        <div class="sign-in">
+          <div class="sign-in__already">
+            <div>Already have an Intuit account?</div>
+            <div><a>I forgot my User ID or Password</a></div>
+          </div>
+          <div class="sign-in__button">
+            <a class="sign-in-button"
+              ><img class="lock-img"
+                src="https://img.icons8.com/material-outlined/24/000000/lock.png"
```
Comment (suretrust) [inline index.html]: The `img` element must have an `alt` attribute. To understand why you can check this [article](https://dev.to/belhassen07/why-is-the-alt-attribute-so-important-for-images-c37).

File: index.html [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+            </div>
+
+            <h1 class="title">Create an Intuit account</h1>
+          </div>
+          <form action="#" enctype="multipart/form-data">
+            <div class="form">
+              <fieldset>
+                <div class="input-form">
+                  <label for="email">Email address</label
+                  ><input type="email" name="email" required />
```
Comment (suretrust) [inline index.html]: The label `for` attribute must contain the input `id` value. For ecample:

`<label for="the-value">Foo</label>`
`<input id="the-value">`

Kindly adjust for all other `label` and `input` elements. You can check [this article on w3schools](https://www.w3schools.com/tags/att_label_for.asp) to see how it was formatted.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+                </div>
+              </fieldset>
+              <fieldset>
+                <div class="input-form">
+                  <label for="phone">Phone (recommended)</label
+                  ><input type="tel" name="phone" />
```
Comment (suretrust) [inline index.html]: Refer to the comment above.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+                </div>
+              </fieldset>
+              <fieldset>
+                <div class="input-form">
+                  <label for="password">Password</label
+                  ><input type="password" name="password" required />
```
Comment (suretrust) [inline index.html]: Refer to the comment above.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+                </div>
+              </fieldset>
+              <fieldset>
+                <div class="input-form">
+                  <label for="confirm">Confirm password</label
+                  ><input type="password" name="confirm" required />
```
Comment (suretrust) [inline index.html]: Refer to the comment above.

**Review by suretrust — CHANGES_REQUESTED** [review]
Hi, @Psiale and @Skyrul,

**Great job thus far guys! :raised_hands:** 

- [ ] Please see my comments to make some adjustments. 

- [ ] Also, the `Create an Intuit account` is supposed to be inside the form tag.

- [ ]  The `One account for everything Intuit.` statement and `Learn more` link are absent in your design.

`[Optional]` The form contents should align to the center as shown in the design you are cloning.

`[Optional]` You can add some text decoration on hover to the links `Legal`, `Privacy`, and `Security`.

**Kindly submit for another review once you've fixed these issues.**

_Happy coding!_ :tada:

File: index.html [hunk 8]
```diff
@@ -0,0 +1,146 @@
...
+      <header>
+        <div class="sign-in">
+          <div class="sign-in__already">
+            <div>Already have an Intuit account?</div>
+            <div><a>I forgot my User ID or Password</a></div>
+          </div>
+          <div class="sign-in__button">
+            <a class="sign-in-button"
+              ><img class="lock-img"
+                src="https://plugin.intuitcdn.net/identity-authn-core-ui/1.2.9/images/icon-whitelock@3x.png"
```
Comment (suretrust) [inline index.html]: The `img` tag still does not have an `alt` attribute.

File: .vscode/settings.json (hunk 1 above)
Comment (suretrust) [inline .vscode/settings.json]: Could you remove this folder?

**Review by suretrust — CHANGES_REQUESTED** [review]
Hi, @Psiale and @Skyrul,

Please check my comments, some of the other corrections are still in your code.

Also, please ensure to fix linter errors before submitting another review.

_Happy coding!_

**Comment by ebukaume** [comment]
## Changes Required! :stop_sign: 

Hi guys :wave: 

This project is almost done, believe me. I like the effort you guys have put in so far. Let's fix these issues:

- [ ] Let's make stickler checks **go green** like @suretrust requested earlier. 
_The error messages are quite explanatory_ . You below and you can see that you typed `input-form` instead of `.input-form` ( _The dot makes is a class, which is what you wanted. Without the dot, the system thinks it is a tag, which it isn't thus the error_ ).
https://github.com/Psiale/microverse-HTML_FORMS/blob/2827dbd698ac40e85523636f317dd2363fae8c71/css/style.css#L59-L63
**Please go through the other errors and fix them all**

-  [ ] Let's make your README descriptive and professional. You can refer to [this guide](https://github.com/othneildrew/Best-README-Template) for inspiration and tips.

**Kindly submit another code review request after fixing the above issues**

Happy Coding :v: 
**Ebuka Umeokonkwo**
 [Github](https://github.com/ebukaume) | [Linkedin](https://www.linkedin.com/in/ebukaume) | [Slack](https://app.slack.com/team/UFDKXE68L) | [Twitter](https://twitter.com/ebukaume)

**Review by KaushikShivam — APPROVED** [review]
# Status: Project Approved! 🎉 

Hello, guys! 👋 
Good job on implementing changes required by the previous TSEs. Everything looks great!!

**Congratulations! You can merge this code to your master branch and move on to the next. All the best**

**If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email.**

Happy coding! 😄 
[Shivam Kaushik](https://github.com/KaushikShivam)
[Github](https://github.com/KaushikShivam) [Twitter](https://twitter.com/kShivamDev) [Linkedin](https://www.linkedin.com/in/shivam-kaushik-bb8162102/)

**Pull request merged**